README.md

```diff
@@ -177,6 +177,10 @@ const twitterDetails = { ... }
 
 For adding emoji 😃 into the texts in `Portfolio.js`, use the `emoji()` function and pass the text you need as an argument. This would help in keeping emojis compatible across different browsers and platforms.
 
+#### Customize Lottie Animations
+
+You can choose a lottie and download it in json format from [this page](https://lottiefiles.com/). In `src/assets/lottie`, replace the lottie json file you want to alter with the same file name. If you want to change the lottie options, go to `src/components/displayLottie/DisplayLottie.js` and change the `defaultOptions` object.
```

Comment:
- *nit*: Lottie should have a capital L
- instead of 'this page' we can use something like 'from sites like this'
- Replacing existing files maybe confusing since iirc they have illustration specific names. Maybe we can rename the Lotties to something like 'landing1', 'landing2' etc. For now we can keep it like this I guess.
- In the last sentence, add a link to the react-lottie docs for more info on the `defaultOptions` object